include/openssl/crypto.h

```diff
@@ -314,6 +314,8 @@ void CRYPTO_mem_debug_realloc(void *addr1, void *addr2, size_t num, int flag,
 void CRYPTO_mem_debug_free(void *addr, int flag,
         const char *file, int line);
 
+int CRYPTO_mem_leaks_cb(int (*cb) (const char *str, size_t len, void *u),
```

Comment:
Gotta document this (man3/OPENSSL_malloc.pod) ?  And does this break the existing functions which are documented?